tests/expose_test.go

```diff
@@ -55,6 +48,26 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 		virtClient, err = kubecli.GetKubevirtClient()
 		tests.PanicOnError(err)
 	})
+	runHelloWorldJob := func(host, port, namespace string) *batchv1.Job {
+		job := tests.NewHelloWorldJob(host, port)
+		job, err := virtClient.BatchV1().Jobs(namespace).Create(job)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
1. I switch to [ExpectWithOffset](https://onsi.github.io/gomega/#making-assertions-in-helper-functions). I hope it helps. 

2. IMO `runHelloWorldJob` doesn't lie and it seems to be better suited as a helper... `createHelloWorldJob` would have side-effects.

Reply:
1. In most of the code, the offset is not correct because originally it may have been at offset n, but later got called by other helpers and became incorrect. The arguments against asserting in helpers have been shared in this discussion, I would just request this in the network area where I mostly need to maintain. Anyway, I will not insist on this at the moment.
2. You claim that "run" translating to "new" and "create" represents well the operation, however, when I read it, I expect it to really run the job, as if something assured it is running.
I do not intend to block on this, but for me, when the function returns it did not run anything, it just created the "what" on the API for some controller to pick it up and then run it.

Reply:
1. Let's ensure that these local helpers will not be called by helpers.